Cargo.toml

```diff
@@ -56,12 +56,18 @@ features = ["alloc", "grab_spare_slice", "rustc_1_40", "rustc_1_55", "serde"]
 [profile.test]
 opt-level = 3
 
+[profile.bench]
+debug = 2
```

Comment:
shouldn't this be debug = 0 to turn off debug info?

Reply:
Quite the opposite. Criterion has `--profile-time` so you can profile your benchmarks. Turning on debug info makes it possible for me to find my way through a `perf report`.